tests/functional/test_download.py

```diff
@@ -1,38 +1,35 @@
 import os
 import re
-import shutil
+import tempfile
 
 from tests.functional.util import run
 
-script_dir = os.path.dirname(__file__)
-
 
 def test_download_file():
-    file = 'EGAF00001753741'
-    download_dir = f'{script_dir}/{file}'
-
-    _assert_successful_run(f'pyega3 -t fetch {file} --output-dir {script_dir}')
-    _assert_complete_files(download_dir)
-    _cleanup(download_dir)
+    file_id = 'EGAF00001753741'
+    with tempfile.TemporaryDirectory() as output_dir:
+        download_dir = f'{output_dir}/{file_id}'
+        run_command_and_assert_download_complete(f'pyega3 -t fetch {file_id} --output-dir {output_dir}')
+        _assert_complete_files(download_dir)
+        cleanup_logs()
 
 
 def test_multipart_downloading():
-    file = 'EGAF00005001625'  # less 200MB, will create 2 slices
+    file_id = 'EGAF00005001625'  # less 200MB, will create 2 slices
```

Comment:
Is the "than" word missing?

Reply:
i updated the comment to be clearer, thanks!